test/suite/helper_tiered.py

```diff
@@ -141,21 +141,46 @@ def gen_tiered_storage_sources(random_prefix='', test_name='', tiered_only=False
         ('s3', dict(is_tiered = True,
             is_tiered_shared = tiered_shared,
             is_local_storage = False,
+            has_cache = True,
             auth_token = get_auth_token('s3_store'),
             bucket = get_bucket_name('s3_store', 0),
             bucket1 = get_bucket_name('s3_store', 1),
-            bucket_prefix = generate_s3_prefix(random_prefix, test_name),
-            bucket_prefix1 = generate_s3_prefix(random_prefix, test_name),
-            bucket_prefix2 = generate_s3_prefix(random_prefix, test_name),
+            bucket_prefix = generate_prefix(random_prefix, test_name),
+            bucket_prefix1 = generate_prefix(random_prefix, test_name),
+            bucket_prefix2 = generate_prefix(random_prefix, test_name),
             num_ops=20,
             ss_name = 's3_store')),
+        ('gcp_store', dict(is_tiered = True,
+            is_tiered_shared = tiered_shared,
+            is_local_storage = False,
+            has_cache = False,
+            auth_token = get_auth_token('gcp_store'),
+            bucket = get_bucket_name('gcp_store', 0),
+            bucket1 = get_bucket_name('gcp_store', 1),
+            bucket_prefix = generate_prefix(random_prefix, test_name),
+            bucket_prefix1 = generate_prefix(random_prefix, test_name),
+            bucket_prefix2 = generate_prefix(random_prefix, test_name),
+            num_ops=100,
+            ss_name = 'gcp_store')),
+        ('azure_store', dict(is_tiered = True,
+            is_tiered_shared = tiered_shared,
+            is_local_storage = False,
+            has_cache = False,
+            auth_token = "",
+            bucket = get_bucket_name('azure_store', 0),
+            bucket1 = get_bucket_name('azure_store', 1),
+            bucket_prefix = generate_prefix(random_prefix, test_name),
+            bucket_prefix1 = generate_prefix(random_prefix, test_name),
+            bucket_prefix2 = generate_prefix(random_prefix, test_name),
+            num_ops=100,
+            ss_name = 'azure_store')),
         ('non_tiered', dict(is_tiered = False)),
     ]
 
     # Return a sublist to use for the tiered test scenarios as last item on list is not a scenario
     # for the tiered tests.  
     if tiered_only:
-        return tiered_storage_sources[:2]
+        return tiered_storage_sources[:4]
```

Comment:
We could make this a little less sensitive to future changes by saying `return tiered_storage_sources[:-1]`, which (I think?) will yield everything except the last item. It would probably also be better to add a comment to the last element in the list saying e.g. "This must be the last item".

Reply:
Done! [0b9a3d0](https://github.com/wiredtiger/wiredtiger/pull/8927/commits/0b9a3d065ebbea92e885c10c3934e94857660dcb)